src/api/posts/test/feeds.test.js

```diff
@@ -75,16 +75,13 @@ describe('Test GET /feeds/delayed endpoint', () => {
 describe('GET /feeds/info', () => {
   test('Should return 200 and valid response object', async () => {
     function checkKeys(resBody) {
-      let bool = true;
       const allKeys = ['waiting', 'active', 'completed', 'failed', 'delayed', 'paused', 'jobCnt'];
-      const resKeys = Object.keys(resBody.queueInfo);
-      for (let i = 0; i < resKeys.length; i += 1) {
-        if (allKeys.indexOf(resKeys[i]) < 0 || typeof resBody.queueInfo[resKeys[i]] !== 'number') {
-          bool = false;
-          break;
+      return Object.keys(resBody.queueInfo).every((key) => {
+        if (!allKeys.includes(key) || typeof resBody.queueInfo[key] !== 'number') {
+          return false;
```

Comment:
```suggestion
      return Object.keys(resBody.queueInfo).every((key) => allKeys.includes(key) && typeof resBody.queueInfo[key] === 'number')
```

This can be shorted. Affirmative is usually easier to read too.

Reply:
Sorry, I missed this part in the previous review.

Reply:
No worries, thank you for the suggestion